ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -56,27 +56,31 @@
   public static final short UNKNOWN_CONTAINER_ID = -1;
 
   /**
-   * The id of {@link #UNKNOWN_PUBLIC_CONTAINER}.
+   * The id for the containers to be associated with the blobs that are put without specifying a target container,
+   * but are specified public. {@link #UNKNOWN_PUBLIC_CONTAINER} is one of the containers that use it.
    */
-  public static final short UNKNOWN_PUBLIC_CONTAINER_ID = -2;
+  public static final short UNKNOWN_PUBLIC_CONTAINER_ID = 0;
```

Comment:
I am inclined to call it `unknown_`, because the `container` information is **unknown** when the blob was put. However, we have the `isPrivate` flag, so instead of assigning `unknown_container`, we can make it a bit specific, that is `unknown_private_` or `unknown_public_` container. What do you think?